tequilapi/endpoints/proposals.go

```diff
@@ -137,3 +155,15 @@ func AddRoutesForProposals(router *httprouter.Router, mc server.Client) {
 	pe := NewProposalsEndpoint(mc)
 	router.GET("/proposals", pe.List)
 }
+
+func (rp *proposalsRes) addFakeQualities() {
+	for i := range rp.Proposals {
+		rp.Proposals[i].Quality = &serviceQuality{
+			Connects: connectsHistory{
+				Success: 100,
```

Comment:
1. No point to mock for all proposals, mock really working proposals only
2. Not point to mock in Desktop artefact, because redeploying real data to desktop machine will be hard. We need to mock in URL, so that we can redeploy easily later with real data